packages/auth/src/providers/cognito/utils/signInHelpers.ts

```diff
@@ -208,7 +277,9 @@ export async function handleSelectMFATypeChallenge({
 }: HandleAuthChallengeRequest): Promise<RespondToAuthChallengeCommandOutput> {
 	const { userPoolId, userPoolClientId, userPoolEndpoint } = config;
 	assertValidationError(
-		challengeResponse === 'TOTP' || challengeResponse === 'SMS',
+		challengeResponse === 'TOTP' ||
+			challengeResponse === 'SMS' ||
+			challengeResponse === 'EMAIL',
```

Comment:
Nit to be more concise:
```suggestion
		['TOTP', 'SMS', 'EMAIL'].includes(challengeResponse),
```